exercises/carmen-salas/short-response.md

```diff
@@ -0,0 +1,16 @@
+# Unit 2 - Lesson 3: Introduction to JavaScript Functions
+## Short Response Questions
+
+1. What is the difference between function parameters and arguments?
+    *Function parameters go in the parenthesis of function which is a place holder for the arguments. Arguments are values inserted in function parameters when the function is called. 
```

Comment:
Great response! One addition:
* Parameters become locally scoped variables inside the body of the function.

One change:
* Instead of saying "values inserted in the function", say "values _passed_ to the function."